src/Utilities/Workloads.js

```diff
@@ -186,12 +188,28 @@ export const flatMapRows = (filteredRows, expandFirst) => {
 };
 
 export const passObjectsFilters = (objects, filters) => {
-  return Object.entries(filters).some(([filterKey, filterValue]) => {
+  console.log(filters);
+  const cleanedUpFilters = _.omitBy(_.cloneDeep(filters), _.isEmpty);
```

Comment:
Please change all the lodash functions imports and don't use `_` since it loads the whole library.

Reply:
True, adjusted that